ftfy/__init__.py

```diff
@@ -221,86 +548,17 @@ def fix_file(
 
     The output is a stream of fixed lines of text.
     """
-    entities = fix_entities
     for line in input_file:
         if isinstance(line, bytes):
             if encoding is None:
                 line, encoding = guess_bytes(line)
             else:
                 line = line.decode(encoding)
-        if fix_entities == 'auto' and '<' in line and '>' in line:
-            entities = False
-        yield fix_text_segment(
-            line,
-            fix_entities=entities,
-            remove_terminal_escapes=remove_terminal_escapes,
-            fix_encoding=fix_encoding,
-            fix_latin_ligatures=fix_latin_ligatures,
-            fix_character_width=fix_character_width,
-            uncurl_quotes=uncurl_quotes,
-            fix_line_breaks=fix_line_breaks,
-            fix_surrogates=fix_surrogates,
-            remove_control_chars=remove_control_chars,
-            remove_bom=remove_bom,
-            normalization=normalization,
-        )
+        if config.unescape_html == "auto" and "<" in line:
+            config = config._replace(unescape_html=False)
```

Comment:
See previous comment about whether it's desirable, or necessary, to change the config here.